todo_app/tests_e2e/test_selenium_e2e.py

```diff
@@ -5,6 +5,10 @@
 import requests
 from dotenv import load_dotenv, find_dotenv
 from selenium import webdriver
+from selenium.webdriver.firefox.service import Service
+from selenium.webdriver.firefox.options import Options
+
+pytest.skip("Skipping... need to fix selenium geckodriver and binary location issues", allow_module_level=True)
```

Comment:
We'll provide some more guidance for potentially running end-to-end tests in Docker as part of M7 - feel free to ask in advance if you're interested